src/utils/common.test.ts

```diff
@@ -10,6 +10,9 @@ describe('CommonUtils', () => {
     it('return types which are not array without any change', () => {
       expect(stripArrayTypeIfPresent('string')).toBe('string');
       expect(stripArrayTypeIfPresent('string []')).toBe('string []');
+      expect(
+        stripArrayTypeIfPresent(undefined as unknown as string),
```

Comment:
Nit: Better to use `ts-expect-error` here instead, so it's clear this is invalid type is intentional. e.g.

```suggestion
        // @ts-expect-error Intentionally testing invalid input
        stripArrayTypeIfPresent(undefined),
```

Reply:
Explained further here: https://github.com/MetaMask/contributor-docs/blob/main/docs/typescript.md#use-ts-expect-error-to-force-runtime-execution-of-a-branch-for-validation-or-testing